smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/ServiceBareBonesClientGenerator.java

```diff
@@ -98,6 +98,7 @@ static String getResolvedConfigTypeName(Symbol symbol) {
     @Override
     public void run() {
         writer.addImport("Client", "__Client", "@aws-sdk/smithy-client");
+        writer.write("export { __Client }\n");
```

Comment:
Was wondering if this should be the following?

```suggestion
        writer.write("export { __Client };");
```